### DISCUSSION TITLE
Silent installation of versions 6.5.2.286 up to 6.5.2-309

### DISCUSSION THREAD

**Discussion by rovabunw** [MuhammedKalkan/OpenLens#161] [General]
Hello,

Up to version 6.5.2 one could use /S /NCRC /ALLUSERS against 
https://github.com/MuhammedKalkan/OpenLens/releases/download/v6.5.2/OpenLens-6.5.2.exe
to install OpenLens-6.5.2.exe into c:\Program Files\OpenLens\OpenLens.exe.

Using the same silent installation switches versions 6.5.2.286, 6.5.2-305, 6.5.2-307 and 6.5.2-309 install into e.g.

C:\Users\Administrator\AppData\Local\Temp\2QnK9XYmgiJGYVdbDbINYx5AgYY\OpenLens.exe

... also the shortcut in the Start Menu is missing. Can you review this please?

**Comment by jkroepke** [comment]
OpenLens-6.5.2-356.exe is an portable exe now. You may look for the setup.exe which is named to OpenLens.Setup.6.5.2-356.exe
